src/cases/store/effects/case-create.effects.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+
+import { Actions } from '@ngrx/effects';
+
+import { hot, cold } from 'jasmine-marbles';
+import { Observable } from 'rxjs/Observable';
+import { empty } from 'rxjs/observable/empty';
+
+import * as fromEffects from './case-create.effects';
+import * as fromActions from '../actions/create-case.action';
+
+export class TestActions extends Actions {
+  constructor() {
+    super(empty());
+  }
+
+  set stream(source: Observable<any>) {
+    this.source = source;
+  }
+}
+
+export function getActions() {
+  return new TestActions();
+}
+
+describe('CaseCreate Effects', () => {
+  let actions$: TestActions;
+  let effects: fromEffects.CaseCreateEffects;
+
+  const toppings = [
```

Comment:
should this be renamed to something more meaningful?

Reply:
good point - this is paste ad copy - will remove thanks